crates/host/src/wasmbus/mod.rs

```diff
@@ -774,156 +802,28 @@ impl Bus for Handler {
             .unwrap_or_default()))
     }
 
-    #[instrument(level = "debug", skip(self))]
+    #[instrument(level = "debug", skip(self, params))]
     async fn call(
         &self,
         target: Option<TargetEntity>,
-        operation: String,
-    ) -> anyhow::Result<(
-        Pin<Box<dyn Future<Output = Result<(), String>> + Send>>,
-        Box<dyn AsyncWrite + Sync + Send + Unpin>,
-        Box<dyn AsyncRead + Sync + Send + Unpin>,
-    )> {
-        let (mut req_r, req_w) = socket_pair()?;
-        let (res_r, mut res_w) = socket_pair()?;
-
-        let links = Arc::clone(&self.links);
-        let aliases = Arc::clone(&self.aliases);
-        let nats = self.nats.clone();
-        let chunk_endpoint = self.chunk_endpoint.clone();
```

Comment:
@rvolosatovs assumption here that wRPC handles chunking or a similar functionality if the message payload is above the NATS limit, is that correct?